charts/newrelic-k8s-metrics-adapter/templates/deployment.yaml

```diff
@@ -18,22 +18,20 @@ spec:
         {{- toYaml .Values.podAnnotations | nindent 8 }}
         {{- end }}
       labels:
-        {{- include "newrelic-k8s-metrics-adapter.labels" . | nindent 8 }}
+        {{- include "common.labels" . | nindent 8 }}
     spec:
-      serviceAccountName: {{ template "newrelic-k8s-metrics-adapter.serviceAccountName" . }}
-      {{- if .Values.podSecurityContext.enabled }}
+      serviceAccountName: {{ include "common.serviceAccount.name" . }}
+      {{- with include "common.securityContext.pod" . }}
       securityContext:
```

Comment:
I see common.securityContext.pod, do we need as well common.securityContext.container?

Reply:
Originally, this chart didn't allow it, do you think we should add it as an extra feature?

Reply:
Yes. The same applies to things like `hostNetwork` and every aspect possible that is in the common library and makes sense.

Reply:
I added it.